docs/glossary.rst

```diff
@@ -0,0 +1,66 @@
+===============================================================================
+XBRL Glossary
+===============================================================================
+
+Axis
+^^^^
+
+Axes are defined in the Taxonomy, and are used as a part of the
+Context to identify a Fact beyond the default
+entity, and Period. Axes can be either typed or explicit. An
+explicit Axis will have all of its allowable values defined in the taxonomy. A
+typed Axis will instead allow a filer to input their own values for the Axis.
+
+Concept
+^^^^^^^
+
+Concepts are defined in the Taxonomy, and describe what sort of Facts can be
+reported in a Filing. They provide a written description, and other metadata that
+the taxonomy creator chooses to include. Not all Concepts will have Facts they are
+directly associated with. This is because Concepts can also be used as a part of a
+Concept tree to define relationships with other Concepts. The root of such a tree
+is a Link Role.
+
+Context
```

Comment:
Is this purely a metadata structure? Or does it correspond to some data structure that would appear in the extracted relational DB?

Reply:
Added explanation about what the `Context` corresponds to in DB